test/hca-rjsf/config/veteranInformation.unit.spec.jsx

```diff
@@ -34,4 +34,46 @@ describe('HCA veteranInformation', () => {
     expect(formDOM.querySelectorAll('input, select').length).to.equal(6);
     expect(formDOM.querySelector('#root_veteranSocialSecurityNumber')).not.to.be.null;
   });
+
+  it('should render demographicInformation page', () => {
```

Comment:
We should have some tests for making sure you get the right number of required field errors when submitting on these pages.